db/src/query.ts

```diff
@@ -125,7 +82,7 @@ class Path {
    * @param x - value for comparison
    * @return - a filter
    */
-  public eq(x: Equatable): EqFilter {
+  public eq(x: dbi.Equatable): EqFilter {
```

Comment:
I'd import Equatable and Comparable

Reply:
Not done: these are internal types, not very important, and we do not modify them here.